modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -312,6 +312,17 @@ private void _validateSingleElementPath(
 				"%s must contain a path parameter between curly braces",
 				"x-must-contain-a-path-parameter-between-curly-braces");
 		}
+
+		Map<String, Serializable> values = objectEntry.getValues();
+
+		if (Objects.equals(values.get("scope"), "group") &&
```

Comment:
I think there is a constant for scope purposes in `ObjectDefinitionConstants`

Reply:
I think that there is a `HeadlessBuilderContants`. Maybe we can add them, `id`, `group` even `externalReferenceCode`

Reply:
> I think that there is a `HeadlessBuilderContants`. Maybe we can add them, `id`, `group` even `externalReferenceCode`

Yes! Maybe including it there would be the best option. I updated my comment as I realised the value is only related with Headless Builder, not Objects